### DISCUSSION TITLE
Limit Combobox Size

### DISCUSSION THREAD

**Discussion by BrendanC23** [shadcn-ui/ui#2046] [Q&A]
I'd like to create a combobox with 1000+ items. This is slow to render the list when the combobox is clicked.Is there a way to show only a some items? For example, to display the first 100 items in the list if there is no search string, and the first 100 that match when there is a search string?

I looked at the cmdk docs, but didn't see anything relevant. Is the best option to build a custom filter/display on top of it?

**Comment by onurhan1337** [comment]
You can checkout this discussions, he's created really good example for that. I hope it's good for you.

https://github.com/shadcn-ui/ui/discussions/2074
